src/routes/notification.route.js

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
+import notificationController from '../controllers/notification.controller';
+import protectRoute from '../middlewares/protect-route.middleware';
+import verifyManager from '../middlewares/verify.manager';
+
+const router = Router();
+
+router.get('/', protectRoute, verifyManager.checkIfManagerExist, notificationController.getUserNotifications);
```

Comment:
I think here you can name the route so that it can specify what your task does.

Reply:
The route is named as notification route for am working on notifications.